Makefile

```diff
@@ -47,3 +47,8 @@ tc-worker-env:
 tc-worker:
 	CGO_ENABLED=$(CGO_ENABLE) GOARCH=amd64 go build
 	docker build -t taskcluster/tc-worker -f tc-worker.Dockerfile .
+
+lint:
+	go get github.com/alecthomas/gometalinter
+	gometalinter --install
+	$(GOPATH)/bin/gometalinter --deadline 10m $$(go list ./... | grep -v /vendor/ | sed 's/github.com\/taskcluster\/taskcluster-worker/./')
```

Comment:
I'd say, rather than choose an include list before starting, instead we iterate through the raised issues, and if we come across one that we decide we don't want to report on, explicitly disable it. Otherwise, we might miss some really useful linting without realising it.